test/test_core_extractor.py

```diff
@@ -90,6 +90,7 @@ def test_process_python_test_extract(self):
         """Function to test the complete end to end process of function definition extractor (True True)"""
         dataframe = extractor((os.path.join(self.file_path, "test_resource", "test_repo")), functionstartwith="test_",
                               report_folder=None)
+        dataframe.to_excel("teee.xlsx")
```

Comment:
Was it by mistake? may be can be removed!

Reply:
Yes it's by mistake. Corrected now